src/doc/gui_manual/Operators/OperatorTypes/PICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,196 @@
+.. _Limit Cycle operator:
+
+Limit Cycle operator
+~~~~~~~~~~~~~~~~~~~~
+
+The Limit Cycle Operator allows one to search for closed material lines (limit
+cycles) which are elliptic LCS's. To perform this search one defines a Poincaré
+section where a series of integral curves are advected. In a Limit Cycle, most
+integral curves will return to the Poincaré section at which point their
+integration will be stopped. Some integral curves may not return to the
+Poincaré section. To limit run-a-way integration, the user can control the
+termination criteria.
+
+For the integral curves that return to the Poincaré section a signed return
+distance is calculated. Curves with a return distance below the cycle tolerance
+are considered to be limit cycles. If a curve does not satisfy the tolerance,
+then its return distance is compared to its neighboring integral curves. If a
+zero crossing is found, then a binary search is conducted. The binary search is
+also limited by a maximum number of iterations.
+
+
+Source
+^^^^^^
+
+Source Type
+"""""""""""
+
+The user can seed the integral curves using samples on a line. This line
+defines a Poincaré section used to detect limit cycles. Source options are:
```

Comment:
Does Sphinx support the special `e` character in `Poincare`? This might require special handling in Sphinx.